documentation-site/components/markdown-elements.js

```diff
@@ -110,7 +110,7 @@ export const DocLink = ({children, href}: {children: string, href: string}) => {
     (parts[0] === '' && parts[1] !== '') || !href.includes('http');
   return (
     <Link href={href} prefetch={internal}>
-      <StyledLink href={href} target={internal ? undefined : '_blank'}>
+      <StyledLink href={href} {...(internal ? {target: '_blank'} : {})}>
```

Comment:
Old code same:
```js
if (internal) {
	return {target: undefined}
} else {
	return {target: '_blank'}
}
```

But your code:
```js
if (internal) {
	return {target: '_blank'}
} else {
	return {};
}
```

Reply:
Good catch! I updated the code to be correct now. Thanks!